### PR TITLE
Change logging config to logging.config.dictConfig

### LINKED ISSUE #97: Inject logging dependency into app creation
The logger should be configured from a config file, which will allow us to easily modify behavior in different environments.

### PR DESCRIPTION
**Pull request by JPrevost**
closes #97

### CODE REVIEW

File: kepler/app.py [hunk 1]
```diff
@@ -45,20 +46,6 @@ def handle_unsupported_format(error):
...
-    handler.setLevel(logging.DEBUG)
-
-    # general flask app logger
-    app.logger.addHandler(handler)
-
-    # http logging
-    requests_log = logging.getLogger('requests.packages.urllib3')
-    requests_log.setLevel(logging.DEBUG)
-    requests_log.addHandler(handler)
+    with open('kepler/logging.conf') as f:
```
Comment (gravesm) [inline kepler/app.py]: We should pull the location of the configuration file from an environment variable. I'm fine with having it default to this location, but we may want to be able to use a different config for testing, for example.

Comment (JPrevost) [inline kepler/app.py]: Cool. Will change.

**Comment by JPrevost** [comment]
@gravesm: d933424 addresses code review suggestions.

I considered setting `LOGGING_CONFIG` in `kepler/settings.py` to consolidate all ENV reading in one place, but decided this felt different enough since there was a default value we wanted to provide to keep it here. I could easily be swayed though so let me know if have opinions.

**Comment by gravesm** [comment]
:heavy_check_mark:

**Pull request merged**